src/cal_build/traceExtract_GH.jl

```diff
@@ -173,7 +171,6 @@ function gh_profiles(tele, mjd, chip, expid; n_sub = 100, make_plots = false)
     all_y_prof_deriv = zeros((size(fiber_inds, 1), size(x_bins, 1)))
 
     for ind in 1:size(fiber_inds, 1)
-        fiber_ind = fiber_inds[ind]
```

Comment:
Can you check that this as intended?  Can't tell if you mean to use `fiber_ind` instead of `ind` within the while loop.